src/Tickets/Commerce/Gateways/PayPal/Merchant.php

```diff
@@ -912,6 +912,36 @@ public function get_locale() {
 		 *
 		 * @param string $locale
 		 */
-		return apply_filters( 'tec_tickets_commerce_gateway_paypal_merchant_locale', $locale );
+		$locale = apply_filters( 'tec_tickets_commerce_gateway_paypal_merchant_locale', $locale );
+
+		return $this->validate_locale( $locale );
+	}
+
+	/**
+	 * Validates the locale to make sure it has a valid length.
+	 * If it's not, it will return the default locale.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $locale The Locale code.
+	 *
+	 * @return string
+	 */
+	public function validate_locale( string $locale ) : string {
+		// check if the length is valid.
+		if ( 5 === strlen( $locale ) ) {
+			return $locale;
+		}
+
+		$fallback_locale = 'en_US';
```

Comment:
Can we set this into a class variable?

Reply:
This is only a fallback value, the default one is coming from `get_locale` of core. So, if anywhere else is needed we can use that.